lua/nvim-treesitter/parsers.lua

```diff
@@ -1555,11 +1555,15 @@ end
 ---@return string[]
 function M.available_parsers()
   if vim.fn.executable "tree-sitter" == 1 and vim.fn.executable "node" == 1 then
-    return vim.tbl_keys(M.list)
+    local parsers = vim.tbl_keys(M.list)
```

Comment:
couldn't those two lines be hoisted out of the conditional?

Reply:
yes, done